File: logisim/app/src/main/java/draw/actions/Strings.java
/* Copyright (c) 2010, Carl Burch. License information is located in the
 * logisim.Main source code and at www.cburch.com/logisim/. */

package draw.actions;

import logisim.util.LocaleManager;
import logisim.util.StringGetter;
import logisim.util.StringUtil;

class Strings {
	private static LocaleManager source
		= new LocaleManager("draw");

	public static String get(String key) {
		return source.get(key);
	}
	public static String get(String key, String arg) {
		return StringUtil.format(source.get(key), arg);
	}
	
	public static StringGetter getter(String key) {
		return source.getter(key);
	}
	public static StringGetter getter(String key, String arg) {
		return source.getter(key, arg);
	}
}

File: LogisimPlus/LogisimPlus/draw/actions/Strings.cs
﻿// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// To produce customized conversions, purchase a Premium Edition license:
// https://www.tangiblesoftwaresolutions.com/product-details/java-to-csharp-converter.html
// ====================================================================================================

/* Copyright (c) 2010, Carl Burch. License information is located in the
 * logisim.Main source code and at www.cburch.com/logisim/. */

namespace draw.actions
{
	using LocaleManager = logisim.util.LocaleManager;
	using StringGetter = logisim.util.StringGetter;
	using StringUtil = logisim.util.StringUtil;

	internal class Strings
	{
		private static LocaleManager source = new LocaleManager("draw");

		public static string get(string key)
		{
			return source.get(key);
		}

		public static string get(string key, string arg)
		{
			return StringUtil.format(source.get(key), arg);
		}

		public static StringGetter getter(string key)
		{
			return source.getter(key);
		}

		public static StringGetter getter(string key, string arg)
		{
			return source.getter(key, arg);
		}
	}

}
